packages/loot-core/src/server/api.ts

```diff
@@ -503,7 +505,7 @@ handlers['api/transaction-update'] = withMutation(async function ({
   }
 
   const { diff } = updateTransaction(transactions, { id, ...fields });
-  return handlers['transactions-batch-update'](diff);
+  return handlers['transactions-batch-update'](diff)['updated'];
```

Comment:
breaking change: makes sense to only return updated transactions, not the whole batch-update object of `{added; updated; deleted}`